core/src/test/scala/org/broadinstitute/dsde/rawls/webservice/SubmissionApiServiceSpec.scala

```diff
@@ -419,14 +420,23 @@ class SubmissionApiServiceSpec extends ApiServiceSpec with TableDrivenPropertyCh
     // Build expected SubmissionListResponse objects
     lazy val failedSubmission = getSubmission(submissionResponseWithFailureMode.submissionId)
     lazy val submission = getSubmission(submissionResponseWithoutFailureMode.submissionId)
+
+    // TODO why are these list v vector?  have i done something wrong?  the vector comes straight from the db...
+    def normalizeSubmissionEntities(response: SubmissionListResponse): SubmissionListResponse =
+      response.copy(submissionEntities = response.submissionEntities.map(_.toVector))
```

Comment:
This whole method looks like it is redundant. Obviously you added it for a reason … what was happening?